tests/data_context/test_data_context_test_yaml_config.py

```diff
@@ -242,6 +242,9 @@ def test_checkpoint_store_with_filesystem_store_backend(
     assert not usage_stats_invalid_messages_exist(messages=caplog.messages)
 
 
+@pytest.mark.filterwarnings(
+    "ignore:String run_ids are deprecated*:DeprecationWarning:great_expectations.data_context.types.resource_identifiers"
+)
```

Comment:
Internal deprecation warning that isn't relevant to our tests. Same comment throughout file.